server/catalog/table_options.cpp

```diff
@@ -364,6 +364,12 @@ Result MakeTableOptions(CreateTableRequest&& request, ObjectId database_id,
   options.shardKeys = std::move(request.shardKeys);
   options.columns = std::move(request.columns);
   options.pkColumns = std::move(request.pkColumns);
+  if (options.pkColumns.empty()) {
+    options.pkColumns = {Column::kFakeId};
+    options.columns.push_back(Column{.id = Column::kFakeId,
+                                     .type = Column::kFakeType,
+                                     .name = std::string{Column::kFakeName}});
+  }
```

Comment:
move this to commands/create_table

Reply:
is not needed as table description will not have it